### ISSUE TITLE
release a opendingux port

### ISSUE THREAD

**Issue by furesoft** [hrydgard/ppsspp#12585]
could you port this emulator to opendingux?

**Comment by Panderner** [comment]
Not all games running full speed for all opendingux handheld devices because most opendingux handheld uses ingenic mips CPUs which is uses under 1 ghz and it lacked mips dynarec some handhelds has lacks left analog stick

**Comment by unknownbrackets** [comment]
Perhaps it just compiles using SDL?  Have you tried this?

-[Unknown]

**Comment by Panderner** [comment]
Is the MIPS dynarec for MIPS CPUs are implemented on PPSSPP?

**Comment by unknownbrackets** [comment]
No, there's no complete MIPS jit.  I think there's some of one, and it doesn't really work.

Also, I googled the devices that run this OS, and I'll just mention that the fastest device (at 1Ghz) has a 320x240 screen, which probably isn't ideal for PSP games rendering at 480x272.  I'm not sure this device will motivate anyone to spend the time required to get a MIPS jit working.

-[Unknown]

**Comment by Panderner** [comment]
I mentioned Odroid Go Advance Is Running a ARM Chip and Screen resolution is 480x320 which is almost fit to PSP's Resolution

**Comment by unknownbrackets** [comment]
Yeah, I would hope it would just compile and work on ARM, hard to tell if changes would be needed.

-[Unknown]

**Comment by unknownbrackets** [comment]
Wrong button, sorry.

-[Unknown]

**Comment by pcercuei** [comment]
> Perhaps it just compiles using SDL? Have you tried this?

@unknownbrackets  It does. No need for a specific port.

It doesn't really work though - it sometimes crash when launching a game (even before the loading screen), sometimes it boots fine, but I couldn't go much farther than in-game menus.
